library/src/main/java/com/novoda/downloadmanager/InternalMigrationStatus.java

```diff
@@ -2,7 +2,7 @@
 
 interface InternalMigrationStatus extends MigrationStatus {
 
-    void update(int currentBatch, int numberOfBatches);
+    void migrationComplete();
```

Comment:
`increments` the internal count of the `numberOfCompletedMigrations`

Reply:
I'm open to ideas 😂

Reply:
cecb7bb

Reply:
Waaay better! 💯